### ISSUE TITLE
Pinia support

### ISSUE THREAD

**Issue by cacholong** [mrichar1/jsonapi-vuex#202]
Hello @mrichar1,

Since Vue is recommending to start using Pinia i was wondering if there are any plans to add Pinia support to this great project?

Looking forward to hear from you.

Regards,

Durk

**Comment by mrichar1** [comment]
Yes it's definitely on the todo list We're already only really focused on Vue3 so it's likely we'll drop support for Vue2 and Vuex going forward and switch to Pinia, unless anyone screams.

**Comment by cacholong** [comment]
Thnx for the quick reply and great to hear the focus is on the future. We'll watch the development with anticipation.

**Comment by mrichar1** [comment]
It's been far too long coming, but the project has finally been ported over to pinia!

Due to an existing project in npm the name has been reversed, so it's now [pinia-jsonapi](https://github.com/mrichar1/pinia-jsonapi)

The functionality is almost identical, though there are some differences due to how pinia differs from vuex - the main changes are listed in `PORTING.md` - but everything should be documented in detail in the main docs.

I've preserved the versioning nd history from jsonapi-vuex for sanity, so it's released as `6.0.0` - though it should still be considered as more of a v1 release and there may be bugs/issues not found through the test suite.

If you have any issues/comments please make them on the new repo, rather than adding them here.
